.github/workflows/_linux-build.yml

```diff
@@ -89,7 +89,7 @@ jobs:
 
       - name: Calculate docker image
         id: calculate-docker-image
-        uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
+        uses: pytorch/test-infra/.github/actions/calculate-docker-image@jeanschmidt/calc-dock-img-ecr-login
```

Comment:
I assume this will be updated before merge?
(Same with the other instances below)